### PR TITLE
Group the configs in the oidc-sample-app.properties

### PR DESCRIPTION
**Pull request by kayathiri4**
Fixes https://github.com/wso2-enterprise/asgardeo-product/issues/3884

Final grouping of properties
```consumerKey=<OAuth Client Key>
consumerKey=<OAuth Client Key>
consumerSecret=<OAuth Client Secret>
scope=openid,address,email,profile
#grantType=code
signatureAlgorithm=RS256

callBackURL=http://localhost:8080/oidc-sample-app/oauth2client
postLogoutRedirectURI=http://localhost:8080/oidc-sample-app/index.html
trustedAudience=http://localhost:8080/sample-app

##Asgardeo related configs
issuer=https://localhost:9443/oauth2/token
authorizeEndpoint=https://localhost:9443/oauth2/authorize
logoutEndpoint=https://localhost:9443/oidc/logout
tokenEndpoint=https://localhost:9443/oauth2/token
jwksEndpoint=https://localhost:9443/oauth2/jwks
#sessionIFrameEndpoint=https://localhost:9443/oidc/checksession

#application pages
skipURIs=/oidc-sample-app/index.html
indexPage=index.html
logoutURL=logout
errorPage=/error.jsp

### CODE REVIEW

**Comment by piraveena** [comment]
What if we move 
```
#application pages
skipURIs=/oidc-sample-app/index.html
indexPage=index.html
logoutURL=logout
errorPage=/error.jsp
```
to bottom, since the user does not need to do anything in basic configs

**Comment by piraveena** [comment]
@mefarazath Any suggesstions?

**Comment by mefarazath** [comment]
> What if we move
> 
> ```
> #application pages
> skipURIs=/oidc-sample-app/index.html
> indexPage=index.html
> logoutURL=logout
> errorPage=/error.jsp
> ```
> 
> to bottom, since the user does not need to do anything in basic configs

Will there be any need to change these when a user runs the sample?
If not, we can get rid of this altogether :)

File: io.asgardeo.tomcat.oidc.sample/src/main/resources/oidc-sample-app.properties [hunk 1]
```diff
@@ -17,19 +17,22 @@
...
+callBackURL=http://localhost:8080/oidc-sample-app/oauth2client
+postLogoutRedirectURI=http://localhost:8080/oidc-sample-app/index.html
+trustedAudience=http://localhost:8080/sample-app
+skipURIs=/oidc-sample-app/index.html
+indexPage=index.html
+logoutURL=logout
+errorPage=/error.jsp
+
+#Asgardeo endpoints
 authorizeEndpoint=https://localhost:9443/oauth2/authorize
```
Comment (mefarazath) [inline io.asgardeo.tomcat.oidc.sample/src/main/resources/oidc-sample-app.properties:30]: Let's take issuer to the start of endpoints or even outside of it.

Comment (piraveena) [reply io.asgardeo.tomcat.oidc.sample/src/main/resources/oidc-sample-app.properties:30]: Or we can name it as #Asgardeo endpoints -> #Asgardeo related configs

Comment (piraveena) [reply io.asgardeo.tomcat.oidc.sample/src/main/resources/oidc-sample-app.properties:30]: can we rename it as Asgardeo related configs ->`IdP related configs`

**Review by piraveena — APPROVED** [review]

**Pull request merged**